app/features/board/boardSlice.tsx

```diff
@@ -53,8 +53,14 @@ const boardSlice = createSlice({
       removeCardFromBoard(board, cardId);
     },
     createBoard(state, action) {
-      const { title, numberOfColumns } = action.payload;
+      const { title, numberOfColumns, selectedBoardId } = action.payload;
```

Comment:
This will not work, if I select 4 columns in the drop-down but choose another board with 3, one column will be named as default.
Instead, if another board is selected, number of columns should be used from the selected board.